extensions/wikia/AdEngine/js/provider/taboola.js

```diff
@@ -16,7 +16,20 @@ define('ext.wikia.adEngine.provider.taboola', [
 		readMoreDiv = document.getElementById('RelatedPagesModuleWrapper'),
 		context = adContext.getContext(),
 		isMobile = context.targeting.skin === 'wikiamobile',
-		pageType = context.targeting.pageType;
+		pageType = context.targeting.pageType,
+		recirculationWikis = {
```

Comment:
I checked this list and unfortunately there are some differences between domains and dbNames :-(
`batman` -> `Batman`
`shadowhunters` -> `mortalinstruments`
`dcanimateduniverse` -> `dcanimated`